src/bundle.rs

```diff
@@ -25,6 +25,9 @@ pub trait DynamicBundle {
     /// Invoke a callback on the fields' type IDs, sorted by descending alignment then id
     #[doc(hidden)]
     fn with_ids<T>(&self, f: impl FnOnce(&[TypeId]) -> T) -> T;
+    // /// Invoke a callback on the fields' type IDs, sorted by descending alignment then id
+    // #[doc(hidden)]
+    // fn with_infos<T>(&self, f: impl FnOnce(&[TypeInfo]) -> T) -> T;
```

Comment:
I'd prefer for this and the related commented dead code to be removed.

Reply:
Oops! Didn't realize I'd left that in.